presto-verifier/src/test/java/com/facebook/presto/verifier/TestShadowing.java

```diff
@@ -53,7 +53,7 @@
 
     public TestShadowing()
     {
-        handle = DBI.open(URL);
+        handle = Jdbi.open(URL);
```

Comment:
slightly related: this should better go into `@BeforeClass`.
Also, the value of `CATALOG` is suspicious.

Reply:
I'll leave these alone since it's unrelated to the Jdbi change.

Reply:
Though I agree with your assessment.